components/Forms/FormAddPlaces.js

```diff
@@ -117,6 +121,8 @@ export default function AddPlaces({ getUniqueValues }) {
         isImage={isImage}
         setIsImage={setIsImage}
         setPublicId={setPublicId}
+        isUploading={isUploading}
+        setIsUploading={setIsUploading}
```

Comment:
As mentioned before: Try to avoid  passing state setter functions as props. It's better to encapsulate them within handler functions.